src/components/PrimaryPanes/index.js

```diff
@@ -71,7 +71,7 @@ class PrimaryPanes extends Component {
         onClick={() => this.showPane("outline")}
         key="outline-tab"
       >
-        Outline View
```

Comment:
Oops, can we move these to `assets/panel/debugger.properties` w/ l10n

https://github.com/devtools-html/debugger.html/blob/master/docs/local-development.md#l10n